.github/workflows/ci.yml

```diff
@@ -19,8 +19,20 @@ jobs:
       # Required to create loop devices.
       options: --privileged
     steps:
-    - uses: actions/checkout@v2
-    - name: Run storage tests
+    - uses: actions/checkout@v3
+    - name: Run storage tests not marked as root as vdsm user
+      run: ./automation/tests-storage.sh vdsm -u=vdsm
+  test-storage-root:
+    env:
+      TRAVIS_CI: 1
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/ovirt/vdsm-test-centos-8
+      # Required to create loop devices.
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
I don't want to diverge, all other oVirt projects use v2, Perhaps push separately as a global change everywhere?
If you do want to touch it then I'd suggest to take into account the recent discussion https://lists.ovirt.org/archives/list/devel@ovirt.org/thread/7SEFKOASOATTMO2NK2SBWMOV4CV6LZOS/#WZ3W6BII34CTQXXLBYJB6W6ECCWEGM4J

Reply:
Ok. Then I will then just remove the commit for this patch. I don't see a reason why not to update it globally to v3 if we decide to take the action suggested in the discussion. Seems a nice chance to do so.